content/_components/ebay/actions.md

```diff
@@ -13,108 +13,84 @@ ComponentVersion: 2.0.2
+{% include img.html max-width="100%" url="img/get-item-feed.png" title="Get Item Feed" %}
 
 Details about the TSV_GZIP file could be found here: [Retrieve a daily Item feed file](https://developer.ebay.com/api-docs/buy/static/api-feed.html)
 
 ### Input fields description (if there is any)
 
-__date__ - The date of the feed file you want. The Item feed files are generated every day and there are always 7 daily feed files available.
-
-Format: yyyyMMdd
+* **date** - The date of the feed file you want. The Item feed files are generated every day and there are always 7 daily feed files available.
 
-Requirement: Must be within 3-10 days in the past
+  * **Format:** `yyyyMMdd`
+  * **Requirement:** Must be within 3-10 days in the past
 
-__categoryId__ - An eBay top-level category ID of the items to be returned in the feed file. The list of eBay category IDs changes over time and category IDs are not the same across all the eBay maketplaces. To get a list of the top-level categories for a markeplace, you can use the Taxonomy API getCategoryTree call. This call retrieves the complete category tree for the marketplace. The top-level categories are identified by the categoryTreeNodeLevel field.
+* **categoryId** - An eBay top-level category ID of the items to be returned in the feed file. The list of eBay category IDs changes over time and category IDs are not the same across all the eBay maketplaces. To get a list of the top-level categories for a markeplace, you can use the Taxonomy API getCategoryTree call. This call retrieves the complete category tree for the marketplace. The top-level categories are identified by the categoryTreeNodeLevel field.
 
-For example: "categoryTreeNodeLevel": 1
+  * **For example:** `"categoryTreeNodeLevel": 1`
 
 For details see [Get Categories for Buy APIs](https://developer.ebay.com/api-docs/buy/buy-categories.html).
 
 Restriction: Must be a top-level category
 
-__range__ - This header specifies the range in bytes of the chunks of the gzip file being returned.
-
-Format: bytes=startpos-endpos
-
-For example, the following retrieves the first 10 MBs of the feed file.
-
-Range bytes=0-10485760
-You can use the getItemFeed call to retrieve a daily TSV_GZIP Item feed file. This file contains all the items listed on a specific day and category. Each item is in new condition, offer at a fixed price (Buy It Now); no auctions, and from eBay trusted sellers.
+* **range** - This header specifies the range in bytes of the chunks of the gzip file being returned.
 
-Maximum: 100 MB
+  * **Format:** bytes=startpos-endpos
+  * **For example:** the following retrieves the first 10 MBs of the feed file.
+    * Range bytes=0-10485760
+    * You can use the getItemFeed call to retrieve a daily `TSV_GZIP` Item feed file. This file contains all the items listed on a specific day and category. Each item is in new condition, offer at a fixed price (Buy It Now); no auctions, and from eBay trusted sellers.
```

Comment:
* You can use the `getItemFeed` call to retrieve a daily `TSV_GZIP` Item feed file. This file contains all the items listed on a specific day and category. Each item is in new condition, offered at a fixed price (Buy It Now); no auctions, and from eBay trusted sellers.